docs/src/pages/lab/slider/StepSlider.js

```diff
@@ -24,7 +24,7 @@ class StepSlider extends React.Component {
 
     return (
       <div className={classes.root}>
-        <Slider value={value} min={0} max={6} step={1} onChange={this.handleChange} />
+        <Slider value={value} min={0} max={102} step={10} onChange={this.handleChange} />
```

Comment:
I tried this locally and `max={104}` made visual difference between old and new behavior clearer. I think we should stick with that.